src/scenes/level.js

```diff
@@ -45,8 +45,8 @@ export class LevelScene extends Phaser.Scene {
     autoClickers = [];
     autoClickDps = 0;
 
-    // Character
-    characterData = saveData;
+    // Character (deep copy)
+    characterData = JSON.parse(JSON.stringify(saveData));
```

Comment:
I initially wanted to use:
`characterData = Object.assign({}, saveData);`
But ironically it only works for the top level of data making it somewhat of a shallow copy lol.
aka: `characterData.characterClass` gets wiped but `characterData.TUTORIAL_ISLAND.enemiesKilled.rat` does not